domains/certificates/Query.API/API/ContractService/Clients/WalletClient.cs

```diff
@@ -133,15 +133,6 @@ private void ValidateOwnerAndSubjectMatch(string owner)
                 throw new HttpRequestException("Owner must match subject");
             }
         }
-        else
```

Comment:
Maybe remove if "IsBearerTokenIssuedByB2C()". Because now all tokens should be issued by it ?

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1484/commits/202a1d87d4955a337d121327a5bcf101ad8dfd30